setup.py

```diff
@@ -44,6 +68,6 @@ def read(fname):
         "Intended Audience :: End Users/Desktop",
         "Intended Audience :: Science/Research",
         "Topic :: Scientific/Engineering",
-        "Topic :: Utilities"
+        "Topic :: Utilities",
```

Comment:
The comma after `Utilities` is wrong, no?

Reply:
Comes from this style guide:
[http://google.github.io/styleguide/pyguide.html#341-trailing-commas-in-sequences-of-items](http://google.github.io/styleguide/pyguide.html#341-trailing-commas-in-sequences-of-items)
I find it handy in Python because duplicating or removing a line is made easier. (Except when removing a line from a 2 elements tuple!)
Conversely I find it annoying in Modelica that it yields an exception...
Anyway it is totally optional:
```
In [16]: [
    ...:     1,
    ...:     2,
    ...: ] == [1, 2]
Out[16]: True
```
Idem with tuples and dictionaries.
So I would not mind agreeing on a different syntax.

Reply:
Sounds good, I wasn't aware of that.